packages/core/contracts-ovm/insured-bridge/implementation/BridgeRouter.sol

```diff
@@ -157,11 +157,11 @@ contract BridgeRouter is OVM_CrossDomainEnabled {
         whitelistedToken.bridgePool = _bridgePool;
         // We want to prevent any situation where a token mapping is whitelisted on this contract but not on the
         // corresponding L2 contract.
-        require(depositContract != address(0), "Deposit contract not set");
+        require(depositContract != address(0), "Deposit contract not set"); //TODO: turn this into a modifier
         sendCrossDomainMessage(
             depositContract,
             _l2Gas,
-            abi.encodeWithSignature("whitelistToken(address,address)", _l1Token, whitelistedToken.l2Token)
+            abi.encodeWithSignature("whitelistToken(address,address)", _l1Token, _l2Token)
```

Comment:
Same comment as above, but why this change? Is it gas savings? Code style? I guess it doesn't matter and _l2Token is slightly nicer to read. My thinking was that reading from the new struct `whitelistedToken` was slightly more correct but I don't think it matters. It's either set correctly or not.

Reply:
they are equivalent honestly. I did it just because it's a bit cleaner and I suspect will use a small amount less gas. In any case I've reverted it for your conflict request.